esdb/client.go

```diff
@@ -759,7 +761,9 @@ func readInternal(
 	result, err := streamsClient.Read(ctx, readRequest, callOptions...)
 	if err != nil {
 		defer cancel()
-		return nil, err
+
+		err = client.grpcClient.handleError(handle, headers, trailers, err)
```

Comment:
Our master branch changed, now you no longer have to pass the `headers` parameter anymore.
```suggestion
		err = client.grpcClient.handleError(handle, trailers, err)
```